lib/src/Uncertainty/Model/RandomVectorImplementation.cxx

```diff
@@ -89,6 +89,12 @@ Point RandomVectorImplementation::getRealization() const
   throw NotYetImplementedException(HERE) << "In RandomVectorImplementation::getRealization() const";
 }
 
+/* Fixed value accessor */
+Point RandomVectorImplementation::getFixedValue(const Point & fixedValue) const
+{
+  return fixedValue;
+}
+
```

Comment:
- better name: getFrozenRealization
- maybe its dangerous to provide a default implementation if new classes appear, better throw

Reply:
> * I proposer another name: getFrozenRealization

OK, done.

> * maybe its dangerous to provide a default implementation if new classes appear, better throw
> * If it cannot be hidden at the c++ level, maybe hide it from swig with %ignore

I don't think this is possible because getFrozenRealization gets called recursively as you move down the event tree: at the end, the antecedent (say, a UsualRandomVector) also calls its own getFrozenRealization method, so it cannot throw and must return the argument that was passed down to it.